Simulator/Recorders/Hdf5Recorder.h

```diff
@@ -60,9 +60,7 @@ class Hdf5Recorder : public Recorder {
 
    // TODO: No parameters needed (AllVertices &vertices)
    /// Compile/capture variable history information in every epoch
-   virtual void compileHistories(AllVertices &neurons) override
-   {
-   }
+   virtual void compileHistories(AllVertices &neurons) override;
```

Comment:
Remember to eliminate unnecessary parameters.

Reply:
yeah, I did it